src/__tests__/fixtures/Optimized/uploaded_files.txt

```diff
@@ -0,0 +1,29 @@
+642282308	processed.opt.ciff
```

Comment:
It is unclear to me what `Optimized` means. I think the directory should give a hint of what it is.